net/arp-scan/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=       ARP Scan
+PLUGIN_VERSION=    1.0
```

Comment:
we should start with 0.1 here for devel version until release, but I can fix that post-merge. Are we good to go here @peppelinux  ? :)